### ISSUE TITLE
Alpha-20664 and beyond: NVDA fails to read the body of messages in Thunderbird 78.1.0

### ISSUE THREAD

**Issue by amirsol81** [nvaccess/nvda#11449]
The issue -- which started with Alpha 20664 -- keeps NVDA silent in the body of messages in Thunderbird 78.1.0 as we press Enter to open them. Navigation with Arrow keys is also impossible as NVDA doesn't see the HTML area. Switching to Alpha 20661 resolves the issue.
### Steps to reproduce:
Simply open a message in Thunderbird 78.1.0, and try using Arrow keys or other navigation keys. Even NVDA+Space doesn't work as the HTML content isn't apparently detected.
### Actual behavior:
NVDA remains silent and no reading/navigation features work.
### Expected behavior:
NVDA should interact with the text displayed as with any HTML content.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
Alpha-20664 and 20666, but switching to Alpha 20661 resolves the issue.
#### Windows version:
Windows 10 Pro 64-Bit, version 2004
#### Name and version of other software in use when reproducing the issue:
Thunderbird 78.1.0
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
20661 is the last Alpha which works well.
#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by josephsl** [comment]
CC @JCSTeh, I think this might be regression from removal of old Mozilla code. Thanks.

**Comment by lukaszgo1** [comment]
Can't reproduce with latest NVDA Alpha and Thunderbird Daily 81.

**Comment by jcsteh** [comment]
@amirsol81, are you by any chance using multiple screen readers? For example, might you have started JAWS, started Thunderbird, exited JAWS and then started NVDA? This is no longer going to work with NVDA at all because the legacy workaround mode Mozilla enables for JAWS is no longer supported by NVDA. I'd be surprised if this wasn't already broken in some way before this, though, maybe just less obvious.

Failing that, can you please provide the dev info obtained by pressing NVDA+f1 once you open a message in Thunderbird? Thanks.

**Comment by amirsol81** [comment]
@jcsteh No, JAWS isn't installed on my machine, and I haven't enabled Narrator either.
Should I provide the dev info with my current working Alpha, 20661, or should I do it after installing 20666 which breaks Thunderbird message-reading? And which Logging level should I select in NVDA's General settings window?
Thanks.

**Comment by jcsteh** [comment]
20666 please. Log level should be set to at least info.

**Comment by amirsol81** [comment]
@jcsteh Quite miraculously I can no longer duplicate it myself! This morning it was always duplicable with 20664 and 20666, but now upgrading directly from 20661 to 20666 has resolved it. I'm confused but glad that the issue has been resolved. Please let me know if I should provide more feedback regarding this.
